terminal/command.go

```diff
@@ -579,7 +607,14 @@ func printcontext(state *api.DebuggerState) error {
 		}
 	}
 
-	for i := l - 5; i <= l+5; i++ {
+	max := func(a, b int) int {
```

Comment:
I think we should just inline this, and move it up above line 603, and instead of `l - 5`, just use this same logic to set `l` appropriately to `l-5` or `0`.